docs/file-rules.kts.md

```diff
@@ -0,0 +1,21 @@
+# The `rules.kts` file
+
+The `rules.kts` file allows defining custom policy rules to automatically reviews scan findings and using a rules DSL based on Kotlin.
+
+For each policy violation one can to definine follow-up actions so users can learn how to resolve the found
```

Comment:
For each policy violation, you can define follow-up actions to help users resolve violations by themselves.

Reply:
Done